src/rust-2021/prelude.md

```diff
@@ -31,5 +31,26 @@ It's identical to the current one, except for three new additions:
 - [`std::convert::TryFrom`](https://doc.rust-lang.org/stable/std/convert/trait.TryFrom.html)
 - [`std::iter::FromIterator`](https://doc.rust-lang.org/stable/std/iter/trait.FromIterator.html)
 
-The library team still needs to formally approve these, which will likely happen soon.
-<!-- TODO: hopefully this happens before we publish this -->
+The tracking issue [can be found here](https://github.com/rust-lang/rust/issues/85684).
+
+## Migration
+
+As a part of the 2021 edition a migration lint, `rust_2021_prelude_collisions`, has been added in order to aid in automatic migration of Rust 2018 codebases to Rust 2021.
+
+### Implementation Reference
```

Comment:
This reference is very interesting, but I'm unsure if this is the level of detail appropriate for this guide. Essentially, the guide should be to help those who are upgrading. This *might* be helpful in the case where someone tries to upgrade and they're confused about why the lint is being triggered, but I don't see that as a major concern. 

I'm personally not against this being here, but if we do keep it, I think I'd like to make it clear to the reader that they most likely don't need this information.

Reply:
That's fair! Part of my reasoning for making it this detailed (as I wouldn't have otherwise) is that it was expressed by @nikomatsakis that one of the goals should be for users to be able to understand the steps needed to hand-migrate code if the lint fails (assuming I understood correctly)

Reply:
That's right, I think I would probably try to make it a bit more oriented around *solutions* people might try. I can try later to give an example.

Reply:
Opened https://github.com/jam1garner/edition-guide/pull/1 -- what do you think, @jam1garner or @rylev ?

Reply:
Add a bunch of comments on that PR - it's certainly the right direction! 😊